docs/BTpanel.md

```diff
@@ -0,0 +1,67 @@
+The tutorial environment is Ubuntu 20.0.4 .  
```

Comment:
I think it would be great that we use the [Docusaurus admonitions](https://docusaurus.io/docs/markdown-features/admonitions) to notify readers the tutorial is under the Ubuntu 20.04 LTS